src/Raven.Studio/typescript/components/common/SplashScreen.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import { ComponentMeta } from "@storybook/react";
+import { SplashScreen } from "./SplashScreen";
+import React from "react";
+import { withBootstrap5, withStorybookContexts } from "test/storybookTestUtils";
+
+export default {
+    title: "Bits/SplashScreen",
+    decorators: [withStorybookContexts, withBootstrap5],
+    component: SplashScreen,
+} as ComponentMeta<typeof SplashScreen>;
+
+export function Loading() {
+    return (
+        <>
+            <SplashScreen></SplashScreen>
```

Comment:
If element doesn't have children then you can use short form: 

```
<SplashScreen />
```

Reply:
+ ```<>``` isn't needed as well.

Reply:
Ok, fixed